### DISCUSSION TITLE
Should I install @remix-run/node?

### DISCUSSION THREAD

**Discussion by zkl2333** [remix-run/remix#894] [Q&A] [answered]
https://github.com/remix-run/remix/blob/57949fe0d73f79811b207fb435657da93f5306d6/packages/create-remix/templates/_shared_ts/remix.env.d.ts#L2

**Comment by tascord** [comment, marked as answer]
Hey @zkl2333! Good question!
Per [the docs](https://remix.run/docs/en/v1/other-api/node):

> Most of the time you will never be importing from this package directly as it's used internally by adapters such as @remix-run/express

So, you probably won't need it!
If you need to install globals for things like unit testing, you may need to install it!

If you have anything else you'd like to know please ask away!
Thanks, <3
